packages/node-utils/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@edge-runtime/node-utils",
+  "description": "Helpers for running edge-compliant code in Node.js environment",
```

Comment:
Can you leave a brief README which describes:

- The differences from `@edge-runtime/vm` (or why we need this new package)
- How to use
- How to test